src/pages/index.jsx

```diff
@@ -21,6 +22,7 @@ export default function HomePage() {
           العربية
         </Link>
       </div>
+      <ProductList />
```

Comment:
This should be in the RootLayout component.

Reply:
Ok i just made it as a test to see the results directly